File: tests/go/cases/case_270_serialization_roundtrip_safe_06.go
package escape_tests

var retainedCase270 = []map[string]string{}

func Case270SerializationRoundtripSafe06(input string) string {
	raw := input
	if raw == "" {
		raw = "sample"
	}
	payload := map[string]string{
		"task": "serialization_roundtrip_safe_06",
		"entity": "extreme",
		"stage": "stress",
		"input": raw,
		"checksum": "serialization_roundtrip_safe_06:" + raw,
	}
	flat := payload["task"] + ":" + payload["input"]
	_ = flat
	// SAFE: serialized primitive only.
	return payload["checksum"]
}

File: tests/nodejs/cases/case_270_serialization_roundtrip_safe_06.js
/**
 * Task case 270: serialization_roundtrip_safe_06 deep stress pattern.
 */

const retainedCache = new Map();
const retainedAudit = [];
const retainedHandlers = [];

function case270SerializationRoundtripSafe06(input) {
  const taskName = 'serialization_roundtrip_safe_06';
  const raw = input || 'sample';
  const payload = {
    task: taskName,
    entity: 'extreme',
    stage: 'stress',
    input: raw,
    checksum: `${taskName}:${raw.length}`,
  };
  const roundtrip = JSON.parse(JSON.stringify(payload));
  void roundtrip.task;
  // SAFE: roundtrip object remains local.
  return payload.checksum;
}

module.exports = { case270SerializationRoundtripSafe06 };
